.clang-format

```
@@ -0,0 +1,128 @@
+#
+#  Copyright (c) 2015 - 2022, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+---
+Language:        Cpp
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterClass:      true
+  AfterControlStatement: false
+  AfterEnum:       false
+  AfterFunction:   true
+  AfterNamespace:  true
+  AfterStruct:     false
+  AfterUnion:      false
+  BeforeCatch:     true
+  BeforeElse:      true
+  IndentBraces:    false
+  SplitEmptyFunction: true
+  SplitEmptyRecord: true
+  SplitEmptyNamespace: true
+IndentWidth:     4
+UseTab:          Never
+BinPackArguments:  true
+BinPackParameters: true
+ExperimentalAutoDetectBinPacking: false
+# IncludeIsMainRegex determines the first group. The rest of the groups are
+# ordered by increasing priority. They are processed in the order they are
+# listed below (it may be different from the priority order)
+SortIncludes:    true
```

```suggestion
IncludeBlocks: Regroup
SortIncludes:    true
```
Automatically groups each section of `#include` statements with an empty line between sections